ci/jobs/synthetic-data-ingestion-dev.yml.j2

```diff
@@ -1,20 +1,23 @@
 jobs:
-  - name: synthetic-data-ingestion-dev
-    plan:
-      - aggregate:
-        - get: aws-analytical-dataset-generation
-          trigger: true
-        - get: aws-internal-compute
-          trigger: false
-        - get: aws-sdx
-          trigger: false
-        - get: aws-ingestion
-          trigger: false
-      - .: (( inject meta.plan.terraform-bootstrap ))
-      - .: (( inject meta.plan.terraform-output-internal-compute ))
-      - .: (( inject meta.plan.terraform-output-ingest ))
-      - .: (( inject meta.plan.terraform-output-sdx ))
-      - .: (( inject meta.plan.synthetic-data-ingestion ))
-        config:
-          params:
+- name: synthetic-data-ingestion-dev
+  plan:
+  - aggregate:
+    - get: aws-analytical-dataset-generation
+      trigger: true
+    - get: aws-ingestion
+      trigger: false
+    - get: aws-management-infrastructure
+      trigger: false
+  - .: (( inject meta.plan.create-aws-profiles ))
+    config:
+        params:
             AWS_ACC: "{{accounts['development']}}"
+            AWS_MGMT_DEV_ACC: "{{accounts['management-dev']}}"
```

Comment:
As mentioned in ingest one, I think this env var shouldn't refer to an environment name, but a name which has the need in it instead.

Reply:
Changed as suggested Steve